### PR TITLE
change command/environment scripting to script/env-script  

### PR DESCRIPTION
**Pull request by matthewrmshin**
Plus:
* 1 instance of `post-script` from `post-command scripting`.
* Improve rose-rug-simple `suite.rc`.
* Run `teardown` before the final set of tests under `t/rose-macro/10-trigger-check.t`.

@arjclark @oliver-sanders please review.

### CODE REVIEW

File: t/rose-task-run/04-run-path-empty/suite.rc [hunk 1]
```diff
@@ -24,8 +24,8 @@ install => my_task_1
            submission timeout = PT1M
            execution timeout  = PT1M
     [[install]]
-        command scripting = """rose task-run"""
+        script = """rose task-run"""
```
Comment (arjclark) [inline t/rose-task-run/04-run-path-empty/suite.rc]: As you're refactoring can you remove the """ round this while you're at it

Comment (matthewrmshin) [inline t/rose-task-run/04-run-path-empty/suite.rc]: I think we should do this as a separate pull request. We do have a lot of unnecessary quoted values in our test battery suites.

**Comment by arjclark** [comment]
Looks OK to me. No problems from the test-battery in my environment.

Over to you @oliver-sanders

**Comment by oliver-sanders** [comment]
Look OK, test battery successful.
